Lib/pathlib.py

```diff
@@ -693,11 +687,15 @@ def __ge__(self, other):
     def __class_getitem__(cls, type):
         return cls
 
-    drive = property(attrgetter('_drv'),
-                     doc="""The drive prefix (letter or UNC path), if any.""")
+    @property
```

Comment:
using a c implemented attrgetter is a common technique to make alias properties just a little faster by sidestepping the bytecode usage in the implementation

Reply:
Really ? Benchmarks show it was slower:

With patch: 4.9194673000019975
Without patch: 5.54981769999722

Reply:
```py
from pathlib import Path
import timeit
p = Path('C:\\hello')

print(timeit.timeit(lambda: p.root, number=10000000))
```